moveit_ros/planning_interface/move_group_interface/src/move_group_interface.cpp

```diff
@@ -1184,40 +1184,40 @@ class MoveGroup::MoveGroupImpl
 }
 }
 
-moveit::planning_interface::MoveGroup::MoveGroup(const std::string &group_name, const boost::shared_ptr<tf::Transformer> &tf, const ros::WallDuration &wait_for_servers)
+moveit::planning_interface::MoveGroupInterface::MoveGroupInterface(const std::string &group_name, const boost::shared_ptr<tf::Transformer> &tf, const ros::WallDuration &wait_for_servers)
```

Comment:
Side topic: I wish MoveIt! would wrap its cpp files with the appropriate namespaces, rather than repeat them in the function header over and over. For example, with this new class name the function header above is incredibly long, when instead we could remove the ``moveit::planning_interface::`` part and just have it once at the top of the file. I would support such a style change and am curious what others think.